packages/codeeditor/src/editor.ts

```diff
@@ -667,6 +672,7 @@ export namespace CodeEditor {
     insertSpaces: true,
     matchBrackets: true,
     autoClosingBrackets: true,
+    handlePaste: true,
```

Comment:
Would it be possible to make this optional? This would constitute a [breaking change](https://github.com/jupyterlab/jupyterlab/issues/7289#issuecomment-604118990).

As an extension developer, this is making it difficult to test our changes against master, since we rely on `@jupyterlab/codeeditor` and other upstream JupyterLab packages, but this interface change has not been released yet.

Reply:
@edzkite Not that master will start accumulating breaking changes as we approach the 3.0 release. You probably want to work against one of the other branches, e.g. the 2.1.x branch https://github.com/jupyterlab/jupyterlab/tree/2.1.x

Reply:
Thanks for pointing out the breaking change though, we want to be more careful about properly logging these in the future, and have some plans to help automate this.